x-pack/plugins/security_solution/docs/prebuilt_rules_customization_rfc.md

```diff
@@ -0,0 +1,2862 @@
+  new-target,
+  elastic-index
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  },
+  {
+    ecs: false,
+    name: "my.new.field",
+    type: "keyword"
+  },
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "keyword"
+  },
+  {
+    ecs: true,
+    name: "new.elastic.ip",
+    type: "ip"
+  }
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  },
+  {
+    ecs: false,
+    name: "host.user.id",
+    type: "keyword"
+  },
+  {
+    ecs: true,
+    name: "process.executable",
+    type: "keyword"
+  },
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=2 style="border-right:3px solid black">Keep user changes if element conflicts and remove element removed by user <br><br> 
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (A _ A - remove element)</b></li></ul>
+      And:
+      <ul><li><b>1st elem: (A B C - keep current) </li><li>2nd elem: (A _ B - remove element</b></li></ul>
+      </td>
+      <td>
+      <pre>
+[
+  logs-*,
+  logstash-*
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  cluster_one:logs-*,
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  logs-new-*,
+  logstash-*
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  cluster_one:logs-*,
+]
+        </pre>
+      </td>
+    </tr>
+    <tr>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "unknown"
+  },
+  {
+    ecs: true,
+    name: "process.executable",
+    type: "string"
+  }
+]
+        </pre>
+      </td>
+      <td>
+        <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+      <td style="border-right:3px solid black">
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.action",
+    type: "keyword"
+  },
+  {
+    ecs: true,
+    name: "process.executable.ip",
+    type: "ip"
+  }
+]
+        </pre>
+        </td>
+      <td>
+      <pre>
+[
+  {
+    ecs: true,
+    name: "event.UPDATED",
+    type: "unknown"
+  }
+]
+        </pre>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+##### MITRE ATT&CK framework (`threat` field)
+
+The `threat` field has a specific [schema](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.gen.ts#L250) that can be handled with its own algorithm in case of conflicts, in order to try to obtain the most reasonable merge between versions. 
+
+**TODO: write algorithm specific to this field**
```

Comment:
I would suggest to exclude this from the RFC. A diff algo for this field will probably be the most complex and difficult to write concrete diff algorithm. I'd suggest to try this only after we have a few basic concrete algorithms already implemented.

Reply:
I agree, also it is a bit strange for the users to be modifying name of technique alone. I think of this objects as bundles.

Reply:
Removing whole MITRE section for now